apps/explorer/src/pages/AddressInfoPage/index.tsx

```diff
@@ -194,7 +207,53 @@ const AddressInfoPage = () => {
             value={txNumber ? addApostrophes(txNumber.toFixed(0)) : !txNumberLoading ? 0 : undefined}
           />
           <InfoGrid.Cell label={t('Nb. of assets')} value={totalNbOfAssets} />
-          <InfoGrid.Cell label={t('Address group')} value={addressGroup?.toString()} />
+          <InfoGrid.Cell
+            label={t('Group(s)')}
+            value={
+              isGrouplessAddress || isGroupedAddress ? (
+                <GroupMenu
+                  label={
+                    isGrouplessAddress ? t('All') : `${t('Group {{ number }}', { number: addressGroup?.toString() })}`
```

Comment:
In this case `addressGroup` will always be defined, right? Then the optional chaining seems reduntant cause it suggests that `addressGroup` can be undefined, which would lead to a wrong displayd string

Reply:
I updated this in my PR